apps/src/turtle/turtle.js

```diff
@@ -1563,13 +1563,21 @@ Artist.prototype.step = function (command, values, options) {
       this.visualization.avatar.visible = true;
       break;
     case 'sticker':
+      let size = MAX_STICKER_SIZE;
+
+      if (values[1]) {
```

Comment:
I think you can remove this outer `if` statement. If `values[1]` is 0, then I think we'd actually want to draw a sticker of size zero (i.e. draw nothing). If it's `null`, then the `typeof` check on the next line will catch it and it'll correctly default to `MAX_STICKER_SIZE` .

Reply:
0 is falsy:  https://stackoverflow.com/questions/19839952/all-falsey-values-in-javascript